res/values/strings.xml

```diff
@@ -153,6 +153,9 @@
 <string name="p_ssid">SSID</string>
 <string name="p_ssid_summary">Station type (1..15; 9=Mobile, 10=APRS-IS)</string>
 <string name="p_ssid_entry">Enter the SSID for your station</string>
+<string name="p_units_title">Distance Units</string>
+<string name="p_units">Unit Preference</string>
+<string name="p_units_entry">Select Distance Unit</string>
```

Comment:
I've stopped adding distinct entry dialog titles, as every string needs to be translated into all languages, and the preference title ("Distance Units") can be re-used here.